packages/x-date-pickers-pro/src/SingleInputDateRangeField/SingleInputDateRangeField.interfaces.ts

```diff
@@ -7,24 +7,25 @@ import {
   DateRangeValidationProps,
 } from '../internal/hooks/validation/useDateRangeValidation';
 
-export interface UseSingleInputDateRangeFieldProps<TInputDate, TDate>
-  extends UseFieldInternalProps<DateRange<TInputDate>, DateRange<TDate>, DateRangeValidationError>,
-    Partial<Omit<DateRangeValidationProps<TInputDate, TDate>, 'value'>> {}
+export interface UseSingleInputDateRangeFieldProps<TDate>
+  extends UseFieldInternalProps<DateRange<TDate>, DateRangeValidationError>,
+    Partial<Omit<DateRangeValidationProps<TDate, TDate>, 'value'>> {}
```

Comment:
Or it's waiting from another PR to remove it in `DateRangeValidationProps`?
```suggestion
    Partial<Omit<DateRangeValidationProps<TDate>, 'value'>> {}
```

Reply:
Yes, I removed `TInputDate` from the field codebase in this PR as well to simplify my work.
But I'm waiting for the main PR before touching the rest of the codebase.